src/topology/vector_bundle/basic.lean

```diff
@@ -8,32 +8,24 @@ import analysis.normed_space.bounded_linear_maps
 import topology.fiber_bundle.basic
 
 /-!
-# Topological vector bundles
+# Vector bundles
 
-In this file we define topological vector bundles.
+In this file we define (topological) vector bundles.
 
-Let `B` be the base space. In our formalism, a topological vector bundle is by definition the type
-`bundle.total_space E` where `E : B → Type*` is a function associating to
-`x : B` the fiber over `x`. This type `bundle.total_space E` is just a type synonym for
-`Σ (x : B), E x`, with the interest that one can put another topology than on `Σ (x : B), E x`
-which has the disjoint union topology.
+Let `B` be the base space, let `F` be a normed space over a normed field `R`, and let
+`E : B → Type*` be a `fiber_bundle` with fiber `F`, in which, for each `x`, the fiber `E x` is a
+topological vector space over `R`.
 
-To have a topological vector bundle structure on `bundle.total_space E`, one should
-additionally have the following data:
+To have a vector bundle structure on `bundle.total_space E`, one should additionally have the
+following properties:
 
-* `F` should be a normed space over a normed field `R`;
-* There should be a topology on `bundle.total_space E`, for which the projection to `B` is
-a topological fiber bundle with fiber `F` (in particular, each fiber `E x` is homeomorphic to `F`);
-* For each `x`, the fiber `E x` should be a topological vector space over `R`, and the injection
-from `E x` to `bundle.total_space F E` should be an embedding;
-* There should be a distinguished set of bundle trivializations (which are continuous linear equivs
-in the fibres), the "trivialization atlas"
-* There should be a choice of bundle trivialization at each point, which belongs to this atlas.
+* The bundle trivializations in the trivialization atlas should be continuous linear equivs in the
+fibres;
+* For any two trivializations `e`, `e'` in the atlas the transition function considered as a map
+from `B` into `F →L[R] F` is continuous on `e.base_set ∩ e'.base_set` with respect to the operator
+norm topology on `F →L[R] F`.
```

Comment:
It might be worth mentioning somewhere in this file that the more naive condition where one only requires continuity of the operations is not enough in infinite dimension for good behavior (in particular wrt bundle constructions), and that the right condition is the one you're mentioning here. And maybe give a pointer to the literature, and/or a simple counterexample illustrating this.